.github/workflows/indy_image_build.yaml

```diff
@@ -0,0 +1,143 @@
+##############################################################################################
+#  Copyright Accenture. All Rights Reserved.
+#
+#  SPDX-License-Identifier: Apache-2.0
+##############################################################################################
+
+# Build Indy docker images for Bevel triggered by :
+#  - push on develop branch for latest tags 
+#  - create tag vx.y.z for release tags
+#  - Can be triggered manually
+
+name: Build Indy docker images for Bevel
+
+on:
+  workflow_dispatch:
+      inputs:
+       indy-node-version:
+        description: 'Enter indy-node-Version to build'
+        required: true
+        default: '1.11.0'
+        type: choice
+        options:
+        - '1.11.0'
+        
+  create:
+    tags:
+      - v*
+  push:
+    branches:
+      - develop
+    paths:
+      - 'platforms/hyperledger-indy/images/**'   
+      - '.github/workflows/indy_images_build.yml'
+
+env:
+  REGISTRY: ghcr.io
+  IMAGE_PATH: ghcr.io/${{ github.repository_owner }}
+
+jobs:
+  # on push to develop branch, create docker tags latest
+  indy-latest:
+    if: ${{ startsWith(github.ref, 'refs/heads/develop') }}
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Set up Docker Buildx
+      uses: docker/setup-buildx-action@v1
+
+    - name: Set up QEMU
+      uses: docker/setup-qemu-action@v1
+
+    - name: Login to the container registry
+      uses: docker/login-action@v1
+      with:
+          registry: ${{ env.REGISTRY }}
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+       
+    - name: Build and push bevel-indy-ledger-txn:latest  
+      id: indy-cli-latest
+      uses: docker/build-push-action@v2
+      with:
+        context: platforms/hyperledger-indy/images/indy-cli
+        push: true
+        tags: ${{ env.IMAGE_PATH }}/bevel-indy-ledger-txn:latest            
+    
+    - name: Build and push bevel-indy-node:latest
+      if: github.event.inputs.indy-node-version == ''
+      id: indy-node-latest
+      uses: docker/build-push-action@v2
+      with:
+        context: platforms/hyperledger-indy/images/indy-node
+        push: true
+        tags: ${{ env.IMAGE_PATH }}/bevel-indy-node:latest
+
+    - name: Build and push bevel-indy-node:provided version
+      if: github.event.inputs.indy-node-version != ''
+      id: indy-node-version
+      uses: docker/build-push-action@v2
+      with:
+        context: platforms/hyperledger-indy/images/indy-node
+        push: true
+        build-args: |
+          BUILDTIME-INDY-NODE-VERSION=${{ github.event.inputs.indy-node-version }}
+        tags: ${{ env.IMAGE_PATH }}/bevel-indy-node:latest,${{ env.IMAGE_PATH }}/bevel-indy-node:${{ github.event.inputs.indy-node-version }}    
```

Comment:
indy-key-mgmt is missing in the latest section

Reply:
added.